bzt/modules/gatling.py

```diff
@@ -291,8 +291,10 @@ def startup(self):
 
         if scenario.get('timeout', None) is not None:
             params_for_scala['gatling.http.ahc.requestTimeout'] = int(dehumanize_time(scenario.get('timeout')) * 1000)
-        if scenario.get('keepalive', None) is not None:
-            params_for_scala['gatling.http.ahc.keepAlive'] = scenario.get('keepalive').lower()
+        if scenario.get('keepalive', True):
```

Comment:
Yes, I agree, but
https://github.com/Blazemeter/taurus/blob/master/bzt/modules/jmeter.py#L1326
https://github.com/Blazemeter/taurus/blob/master/bzt/modules/pbench.py#L291
and even
https://github.com/Blazemeter/taurus/blob/master/bzt/modules/ab.py#L103